config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -731,3 +731,49 @@ periodics:
         limits:
           cpu: 1
           memory: "2Gi"
+- name: ci-kubernetes-e2e-gce-network-metric-measurement
+  tags:
+    - "perfDashPrefix: network-performance"
+    - "perfDashJobType: performance"
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-periodics: "true"
+    preset-e2e-scalability-periodics-master: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-network
+    testgrid-tab-name: metric-measurement
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20201111-a263fd7-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=60
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/latest
+      - --gcp-node-image=gci
+      - --gcp-nodes=102
+      - --provider=gce
+      - --test=false
+      - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+      - --test-cmd-args=cluster-loader2
+      - --test-cmd-args=--nodes=100
+      - --test-cmd-args=--provider=gce
+      - --test-cmd-args=--report-dir=/workspace/_artifacts
+      - --test-cmd-args=--testsuite=testing/network/suite.yaml
+      - --test-cmd-name=ClusterLoaderV2
+      - --timeout=40m
+      - --use-logexporter
+      resources:
+        requests:
+          cpu: "500m"
```

Comment:
Please revert this to 1cpu as it was before.

This isn't the number I was talking about - what you need to change is the actual worker pods resources, i.e.:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml#L25

Reply:
BTW - when you will be changing it - I just pushed the image, so please update the image too:
```
0.1: digest: sha256:4e8a2c2df8c485b0d58f4815f3b99782aa4e379c07e122b8f96fa9709c826f3a size: 3251
Pushed gcr.io/k8s-testimages/netperfbenchmark with :0.1 tag
```

Reply:
Sorry was an oversight, have updated the cpu resources and image name in below PR :
https://github.com/kubernetes/perf-tests/pull/1781